source/base/Text.ooc

```diff
@@ -53,6 +53,30 @@ Text: cover {
 	operator != (other: String) -> Bool { !(this == other) }
 	operator != (other: This) -> Bool { !(this == other) }
 	operator + (other: This) -> This { This new(this _buffer + other _buffer) }
+	operator + (other: Int) -> This {
+		converted := other toString()
```

Comment:
Did you actually run memtest that time, or were you just lucky, and the memory your pointer pointed to still contained the correct data, even if it was, in fact, invalid?